<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JuickSettings</class>
 <widget class="QWidget" name="JuickSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="user_label">
     <property name="text">
      <string>@user_id</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="user_layout">
     <item>
      <widget class="QCheckBox" name="user_under">
       <property name="text">
        <string>Underline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="user_italic">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="user_bold">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="post_label">
     <property name="text">
      <string>#post_id</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="post_layout">
     <item>
      <widget class="QCheckBox" name="post_under">
       <property name="text">
        <string>Underline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="post_italic">
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="post_bold">
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="extra_cmds">
     <property name="text">
      <string>Show additional commands</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>User commands</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="user_line_edit">
     <property name="toolTip">
      <string>Commands separated by space</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Post commands</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="post_line_edit">
     <property name="toolTip">
      <string>Commands separated by space</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>extra_cmds</sender>
   <signal>toggled(bool)</signal>
   <receiver>user_line_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extra_cmds</sender>
   <signal>toggled(bool)</signal>
   <receiver>post_line_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
